preprocessing_qc/process_tcga.ipynb

```diff
@@ -0,0 +1,915 @@
+{
```

Comment:
<p>you could add a bit more description of what are we looking at/where data comes from and is processed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_atac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>added in <a href="https://github.com/greenelab/buddi_atac/pull/1/commits/4cfd7789337a8585751a287d95bc47e48139594a" target="_blank">4cfd778</a></p>